src/app/portlets/dot-edit-page/main/dot-edit-page-nav/dot-edit-page-nav.component.ts

```diff
@@ -26,11 +27,19 @@ interface DotEditPageNavItem {
 })
 export class DotEditPageNavComponent implements OnChanges {
     @Input() pageState: DotRenderedPageState;
+    @Output() buttonClicked: EventEmitter<{ inode: string; label: string }> = new EventEmitter();
     model: Observable<DotEditPageNavItem[]>;
     isEnterpriseLicense: boolean;
 
     constructor(private dotLicenseService: DotLicenseService, public dotMessageService: DotMessageService, public route: ActivatedRoute) {}
 
+    emitClick($event: MouseEvent, inode: string, label: string): void {
+        $event.stopPropagation();
+        if (inode) {
```

Comment:
I think we should emit the event anyway and check the inode into 'dot-edit-page-main.component'